CMakeLists.txt

```diff
@@ -321,17 +322,16 @@ source_group("world" FILES ${world_sources})
 include_directories(${XLEngine_SOURCE_DIR}/thirdparty/AngelCode/sdk/angelscript/include)
 
 # Include/library directories.
-# - TODO: find ENet. Currently has to be found and set manually.
-# - TODO: Use enet64.lib on 64-bit.
 set(xlengine_include_dirs
+    ${ENET_INCLUDE_DIR}
     ${GLEW_INCLUDE_DIR}
     ${IL_INCLUDE_DIR})
 set(xlengine_libraries
+    angelscript
+    ${ENET_LIBRARY}
```

Comment:
These should be `ENET_INCLUDE_DIRS` and `ENET_LIBRARIES`.

Reply:
Done. I looked up the DevIL convention [here](https://cmake.org/cmake/help/v3.0/module/FindDevIL.html) and it only has `IL_INCLUDE_DIR` (singular).

Reply:
Yeah, it seems inconsistent between the various `Find*` scripts, unfortunately. In this case the singular names are the cache entries that are either set appropriately or are some variation of `-NOTFOUND`, whereas the plural names are not cached but are set as long as `ENET_FOUND` is true or are left empty otherwise.